packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -60,7 +60,7 @@ export function UserHeaderItem(props: UserHeaderItemProps): React.ReactElement {
     if (loading || !data)
         return (
             <LoadingWrapper>
-                <Loading behavior="fillParent" size={20} color="inherit" />
+                <Loading behavior="fillParent" color="inherit" />
```

Comment:
TODO: size

Reply:
@jamesricky `size` seems to be useful. It seems bothersome to always create a styled component to adjust the size. 
Should we (re)implement the size prop for `Loading`? What do you think?